internal/indexer/definitions/sugoimusic.yaml

```
@@ -0,0 +1,80 @@
+#id: sugoimusic
+name: SugoiMusic
+identifier: sugoimusic
+description: SugoiMusic (SM) is a private torrent tracker for MUSIC
+language: en-us
+urls:
+  - https://sugoimusic.me/
+privacy: private
+protocol: torrent
+supports:
+  - irc
+source: gazelle
+settings:
+  - name: authkey
+    type: secret
+    required: true
+    label: Auth key
+    help: Right click DL on a torrent and get the authkey.
+
+  - name: torrent_pass
+    type: secret
+    required: true
+    label: Torrent pass
+    help: Right click DL on a torrent and get the torrent_pass.
+
+irc:
+  network: SugoiMusic
+  server: irc.sugoimusic.me
+  port: 6697
+  tls: true
+  channels:
+    - "#sugoi-announce"
+  announcers:
+    - Yuki
+  settings:
+    - name: nick
+      type: text
+      required: true
+      label: Nick
+      help: Bot nick. Eg. user-autodl
+
+    - name: auth.account
+      type: text
+      required: false
+      label: NickServ Account
+      help: NickServ account. Make sure to group your user and bot.
+
+    - name: auth.password
+      type: secret
+      required: false
+      label: NickServ Password
+      help: NickServ password
+
+    - name: invite_command
+      type: secret
+      default: "Yuki enter #sugoi-announce USERNAME IRCKEY"
+      required: true
+      label: Invite command
+      help: Invite auth with Yuki. Replace USERNAME and IRCKEY.
+
+  parse:
+    type: single
+    lines:
+      - test:
+          - "[Album] Artist – Album Name [FLAC / 2017 / catnumber] - https://sugoimusic.me/torrents.php?id=000000 / https://sugoimusic.me/torrents.php?action=download&id=000000 - japanese,pop,female.vocalist Uploaded by: Uploader"
+          - "[Single] Artist – Album Name [MP3] - https://sugoimusic.me/torrents.php?id=000000 / https://sugoimusic.me/torrents.php?action=download&id=000000 - japanese,pop,dance Uploaded by: Uploader"
+        pattern: '(\[(.*)\] .* \[(.*)\]) - .*id=(.*) \/ (https?://.+/).+id=(\d+) - (.*) Uploaded by: (.*)'
```

```suggestion
        pattern: '(\[(.*)\] .*? – .+? \[((?:.*?)?(?: \/ )?(\d{4})?(?: \/ )?\w+)\]) - https:\/\/sugoimusic\.me\/torrents\.php\?id=(\d+) \/ https:\/\/sugoimusic\.me\/torrents\.php\?action=download&id=(\d+) - (.*) Uploaded by: (.*)'
```

This should capture the optional `year` separately as well as a `releaseTag`

https://regex101.com/r/PCwvbv/1

---

This looks great, thank you.

---

This looks great, thank you.